### PR TITLE
Editing domain

### PR DESCRIPTION
**Pull request by Buzzardo**

### CODE REVIEW

File: spring-batch-docs/asciidoc/domain.adoc [hunk 1]
```diff
@@ -765,4 +767,3 @@ As long as the batch namespace has been declared, any of its
 elements can be used. More information on configuring a
 Job can be found in <<job.adoc#configureJob,Configuring and Running a Job>>. More information on configuring a `Step` can be
 found in <<step.adoc#configureStep,Configuring a Step>>.
-
```
Comment (mminella) [inline spring-batch-docs/asciidoc/domain.adoc]: We leave the blank line at the end because it makes diffs nicer with git.

Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: Interesting. My file has a blank line at the end. I wonder why it didn't make it into Github.

File: spring-batch-docs/asciidoc/domain.adoc [hunk 2]
```diff
@@ -11,67 +11,67 @@ include::toggle.adoc[]
...
+file or Java-based configuration. This configuration may be referred to as
 the "job configuration". However, `Job` is just the
-top of an overall hierarchy:
+top of an overall hierarchy, as shown in the following diagram:
 
 .Job Hierarchy
 image::{batch-asciidoc}images/job-heirarchy.png[Job Hierarchy, scaledwidth="60%"]
 
-In Spring Batch, a Job is simply a container for Steps. It combines
+In Spring Batch, a `Job` is simply a container for `Step` items. It combines
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: "items", then. Thanks.

File: spring-batch-docs/asciidoc/domain.adoc [hunk 3]
```diff
@@ -11,67 +11,67 @@ include::toggle.adoc[]
...
 multiple steps that belong logically together in a flow and allows for
 configuration of properties global to all steps, such as restartability.
 The job configuration contains:
 
-* The simple name of the job
-* Definition and ordering of Steps
-* Whether or not the job is restartable
+* The simple name of the job.
+* Definition and ordering of Step items.
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: Yup. Please do.

File: spring-batch-docs/asciidoc/domain.adoc [hunk 4]
```diff
@@ -102,12 +102,13 @@ Instead, the `<job>` tag can be used:
 ----
 endif::backend-html5[]
 
+// TODO Why different content for HTML versus PDF?
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: Righto. Feel free to remove the comment. Thanks.

File: spring-batch-docs/asciidoc/domain.adoc [hunk 5]
```diff
@@ -196,86 +197,85 @@ allows a developer to effectively control how a
...
 |===
 |Property |Definition
-|status
+|Status
 |A `BatchStatus` object that
-indicates the status of the execution. While running, it's
-BatchStatus.STARTED, if it fails, it's BatchStatus.FAILED, and
-if it finishes successfully, it's BatchStatus.COMPLETED
+indicates the status of the execution. While running, it is
+BatchStatus#STARTED. If it fails, it is BatchStatus.FAILED.
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: Yup. Please add those characters. Thanks.

File: spring-batch-docs/asciidoc/domain.adoc [hunk 6]
```diff
@@ -196,86 +197,85 @@ allows a developer to effectively control how a
...
+If it finishes successfully, it is BatchStatus.COMPLETED
```
[the remark this answers is not in the document]
Comment (Buzzardo) [reply spring-batch-docs/asciidoc/domain.adoc]: Yes, please. Sorry I missed it.

**Comment by mminella** [comment]
Polished and merged as 3be6d9df66642613ffa3995f2c7b84ef884f7f10

**Pull request closed without merging**